Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -98,7 +119,22 @@ class CartViewModel: BaseViewModel {
         return cart.value?.lineItems?[indexPath.row].totalPrice?.description ?? "N/A"
     }
 
-    // MARK: - Commercetools product projections querying
+    private func deleteLineItemAtIndexPath(indexPath: NSIndexPath) {
+        if let cartId = cart.value?.id, version = cart.value?.version, lineItemId = cart.value?.lineItems?[indexPath.row].id {
+            self.isLoading.value = true
+            Commercetools.Cart.update(cartId, version: version, actions: [["action": "removeLineItem",
+                                                                           "lineItemId": lineItemId]], result: { result in
+                if let errors = result.errors where result.isFailure {
+                    super.alertMessageObserver.sendNext(self.alertMessageForErrors(errors))
+                    self.isLoading.value = false
+                }
+                self.queryForActiveCart()
```

Comment:
`self.queryForActiveCart()` also performs a call for running recalculation on the API side before retrieving the new "recalculated" instance.